library/nats-streaming

```diff
@@ -1,28 +1,28 @@
 Maintainers: Ivan Kozlovic <ivan@synadia.com> (@kozlovic)
 GitRepo: https://github.com/nats-io/nats-streaming-docker.git
-GitCommit: eaff03b1e4f88d0eb0eb476a8d1b36cac6fdf8df
+GitCommit: 5639c0377daecb06d162641d81c830fa5a204c00
 
 Tags: 0.9.2-linux, linux
 SharedTags: 0.9.2, latest
```

Comment:
If this is now 0.10, shouldn't the tags change to reflect that?

Reply:
Oh right, let me update the PR.. sorry..